Index: binarySearch/src/haybales.java
===================================================================
diff --git a/binarySearch/src/haybales.java b/binarySearch/src/haybales.java
deleted file mode 100644
--- a/binarySearch/src/haybales.java	(revision 6b85d359c5746b7e56ff81c75a1d12da1b525e6e)
+++ /dev/null	(revision 6b85d359c5746b7e56ff81c75a1d12da1b525e6e)
@@ -1,2 +0,0 @@
-public class haybales {
-}
Index: binarySearch/src/cowdance.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/binarySearch/src/cowdance.java b/binarySearch/src/cowdance.java
new file mode 100644
--- /dev/null	(date 1641250110746)
+++ b/binarySearch/src/cowdance.java	(date 1641250110746)
@@ -0,0 +1,18 @@
+import java.io.*;
+import java.util.*;
+
+public class cowdance {
+    public static void main(String[] args) throws Exception {
+        BufferedReader b = new BufferedReader(new FileReader("cowdance.in"));
+        PrintWriter p = new PrintWriter(new BufferedWriter(new FileWriter("cowdance.out")));
+
+        StringTokenizer st = new StringTokenizer(b.readLine());
+        int n = Integer.parseInt(st.nextToken());
+        int t = Integer.parseInt(st.nextToken());
+        int[] times = new int[n];
+        for (int i = 0; i < n; i++) {
+            times[i] = Integer.parseInt(b.readLine());
+        }
+
+    }
+}
diff --git a/binarySearch/haybales.in b/binarySearch/cowdance.in
rename from binarySearch/haybales.in
rename to binarySearch/cowdance.in
diff --git a/binarySearch/haybales.out b/binarySearch/cowdance.out
rename from binarySearch/haybales.out
rename to binarySearch/cowdance.out
